src/client/Microsoft.Identity.Client/WwwAuthenticateParameters.cs

```diff
@@ -34,13 +34,27 @@ public class WwwAuthenticateParameters
         /// Resource for which to request scopes.
         /// This is the App ID URI of the API that returned the WWW-Authenticate header.
         /// </summary>
+        /// <remarks>
+        /// Clients that perform resource validation (e.g. by comparing the host part of the resource against a list of known good hosts), 
```

Comment:
Should this be added as a code example to the [docs ](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/WWW-Authenticate-parameters#how-to---use-the-wwwauthenticateparameters-class)too?